### PR TITLE
[cli] fallback to _start value for --exportStart if argument was skipped

### PR DESCRIPTION
**Pull request by MaxGraey**
- [x] I've read the contributing guidelines
- [x] I've added my name and email to the NOTICE file

### CODE REVIEW

**Comment by dcodeIO** [comment]
Isn't unquoting something the shell / Node already does according to conventions? Seems to me that postprocessing again can lead to strange issues, in particular if we ever add an option that may depend on inner quotes in strings. Do you have an example of what doesn't work currently and would be fixed?

**Comment by MaxGraey** [comment]
`util/options.js` doesn't do this. Does it better add unquotting to `options.js`?

**Comment by dcodeIO** [comment]
I think it also doesn't need to, because if one specifies `"abc"` on the command line, the shell / Node.js already remove the quotes according to their conventions, so the CLI only sees `abc` anyway.

**Comment by dcodeIO** [comment]
I think there one wouldn't specify `"\"hello\""`, but just put `"hello"` so that processing the JS/JSON does its natural thing.

File: cli/index.js [hunk 1]
```diff
@@ -309,7 +309,13 @@ export async function main(argv, options) {
...
   assemblyscript.setImportTable(compilerOptions, opts.importTable);
   assemblyscript.setExportTable(compilerOptions, opts.exportTable);
-  assemblyscript.setExportStart(compilerOptions, typeof opts.exportStart === "string" ? opts.exportStart : null);
+  if (opts.exportStart) {
+    if (typeof opts.exportStart === "string" && opts.exportStart !== "") {
+      assemblyscript.setExportStart(compilerOptions, opts.exportStart);
+    } else {
+      return prepareResult(Error("exportStart option required string argument"));
+    }
+  }
```
Comment (dcodeIO) [inline cli/index.js:318]: An alternative would be to assume `_start` as the default if just `--exportStart` is given, effectively making the name optional. Would that preferable over an error, also given that it then behaves like `--explicitStart` did before, just with a different name?

Comment (MaxGraey) [reply cli/index.js:318]: okey!

**Review by dcodeIO — APPROVED** [review]

**Pull request merged**